cmd/lakefs/cmd/migrate.go

```diff
@@ -56,14 +69,18 @@ var gotoCmd = &cobra.Command{
 	Run: func(cmd *cobra.Command, args []string) {
 		cfg := loadConfig()
 		ctx := cmd.Context()
+		dbParams := cfg.GetDatabaseParams()
+		if dbParams.KVEnabled {
+			fmt.Printf("Unsupported command for KV migration version\n")
+			os.Exit(1)
+		}
```

Comment:
Why is that unsupported?
I understand that the first version with KV cannot have a goto version, but is it correct for KV in general?

Reply:
Exactly what you said - this is currently unsupported. Once we define what it means to perform db migration in the KV world and if we even support the goto command - we will change this accordingly